python/pyspark/sql/streaming/proto/StateMessage_pb2.pyi

```diff
@@ -54,7 +54,7 @@ class _HandleState:
 class _HandleStateEnumTypeWrapper(
     google.protobuf.internal.enum_type_wrapper._EnumTypeWrapper[_HandleState.ValueType],
     builtins.type,
-):  # noqa: F821
+):
```

Comment:
It seems that `# noqa: F821` on this line shouldn't be deleted. 
Is the environment different from GA? I 

https://github.com/apache/spark/blob/58c77bab93173969e37fd5dd4619f514816e1196/.github/workflows/build_and_test.yml#L727-L730

installed the following Python  3.11 modules in the newly generated execution environment on my local machine:

```
Package           Version
----------------- ---------------
black             23.9.1
click             8.1.7
mypy              1.8.0
mypy-extensions   1.0.0
mypy-protobuf     3.3.0
packaging         24.1
pathspec          0.12.1
pip               24.3.1
platformdirs      4.3.6
protobuf          5.28.3
setuptools        74.1.2
types-protobuf    5.28.3.20241030
typing_extensions 4.12.2
```

Reply:
Thanks @LuciferYang, let me play around with my local env. The `StateMessage_pb2.pyi` in the PR is exactly the same as the script generated, I guess it might be related to local env.

Reply:
@LuciferYang That's because of my local env, mine is using `mypy-protobuf` with `3.4.0`, maybe that's the cause here. It's green now after I updated my local env, thanks for the help!